pynndescent/pynndescent_.py

```diff
@@ -1704,27 +1709,116 @@ def query(self, query_data, k=10, epsilon=0.1):
 
         return indices, dists
 
-    def update(self, X):
+    def update(self, xs_updated=None, updated_indices=None, xs_fresh=None):
+        """
```

Comment:
Following sklearn conventions (I know this class isn't an an sklearn estimator, but still) the general use of update is to take only new data (as used to be thecase). If you were to simply reorder the args here so that xs_fresh were first then this interface would be preserved.
```suggestion
    def update(self, xs_fresh=None, xs_updated=None, updated_indices=None):
        """
```